### PR TITLE
added nav and styled scroll

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: pages/docs/docs.html [hunk 1]
```diff
@@ -475,6 +477,34 @@ <h1>Modal heading</h1>
                 style="width: 846px; height: 356px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
             </iframe>
         </div>
+        <iframe width="0" height="0" id="navigation"></iframe>
```
Comment (devananth) [inline pages/docs/docs.html:480]: I couldn't get , why you are adding an extra `iframe` tag ?

File: pages/docs/docs.html [hunk 2]
```diff
@@ -475,6 +477,34 @@ <h1>Modal heading</h1>
...
+            <text class="title"> Navigation</text>
+            <text class="subtext"> Navigation bar is used to navigate through the web application with ease and items in navigation bar changes
+                depending upon the screen size.
+            </text>
+            <text class="example"> Example</text>
+
+            <div class="example-box">
+                <div class="container-nav">
+                    <div class="flex">
+                        <img alt="logo" width="50px" src="../../assets/bg-teal.png">
```
Comment (devananth) [inline pages/docs/docs.html]: I think we should not use `inline styles` , better have a class `.logo` and apply styles there.

File: components/navigation.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    align-items: center;
+}
+.nav-options>*{
+    padding: 0.5rem 1rem;
+    border-radius: 4px;
+    text-decoration: none;
+    color: var(--white);
+}
+.brandname{
+    margin: 0px 0px 0px 20px;
```
Comment (devananth) [inline components/navigation.css]: I think this can be written in this way `margin-left : 20px` , right ?

**Review by devananth — CHANGES_REQUESTED** [review]

File: components/navigation.css (hunk 3 above)
Comment (ajay-mandaviya) [inline components/navigation.css]: just a suggestion no need to change ,here you can also used direct  a margin left 20px or  margin 0 0 0 20px,

File: pages/docs/docs.html (hunk 1 above)
Comment (shivanigangadharan) [reply pages/docs/docs.html:480]: This is for the next component (which is hidden a bit due to navbar), so giving the iframe with id, and that is where the links on the sidebar are pointed to.

**Review by ajay-mandaviya — APPROVED** [review]
all good

File: components/navigation.css (hunk 3 above)
Comment (shivanigangadharan) [reply components/navigation.css]: Done in [cfaba63](https://github.com/shivanigangadharan/style-ui/pull/22/commits/cfaba633c1e1420ab8b7534a40286578403b03b3).

Comment (shivanigangadharan) [reply components/navigation.css]: Done [cfaba63](https://github.com/shivanigangadharan/style-ui/pull/22/commits/cfaba633c1e1420ab8b7534a40286578403b03b3).

File: pages/docs/docs.html (hunk 2 above)
Comment (shivanigangadharan) [reply pages/docs/docs.html]: Done in commit [cfaba63](https://github.com/shivanigangadharan/style-ui/pull/22/commits/cfaba633c1e1420ab8b7534a40286578403b03b3).

**Review by devananth — APPROVED** [review]
You can merge it

**Pull request merged**